mammoth_snowplow/src/mammoth_snowplow.cpp

```diff
@@ -0,0 +1,10 @@
+#include <cstdio>
+
+int main(int argc, char ** argv)
+{
+  (void) argc;
+  (void) argv;
+
+  printf("hello world mammoth_snowplow package\n");
+  return 0;
+}
```

Comment:
is there any plan for this file?

Reply:
nope lol. i was in the process of fixing it then had to eat dinner